### PR TITLE
Hardening

### PR DESCRIPTION
**Pull request by DTadrzak**
Few changes in configuration files.
@pawelsocha 
@ss7pro 
@PiotrProkop

### CODE REVIEW

File: config/neutron/templates/dnsmasq.conf.j2 [hunk 1]
```diff
@@ -1 +1 @@
-log-facility=-
+log-facility=/tmp/dnsmasq.log
```
Comment (ss7pro) [inline config/neutron/templates/dnsmasq.conf.j2]: We don't want that change.

File: config/neutron/templates/ml2_conf.ini.j2 [hunk 2]
```diff
@@ -16,11 +16,12 @@ vni_ranges = 1:1000
...
 
 [securitygroup]
-firewall_driver = neutron.agent.linux.iptables_firewall.OVSHybridIptablesFirewallDriver
+firewall_driver = noop
+enable_security_group = False
 
 [agent]
 tunnel_types = vxlan
-l2_population = true
+l2_population = false
```
Comment (ss7pro) [inline config/neutron/templates/ml2_conf.ini.j2]: Please add here a "#FIXME" comment here.

File: config/neutron/templates/ml2_conf.openvswitch.ini.j2 [hunk 3]
```diff
@@ -4,13 +4,17 @@ debug = true
 mechanism_drivers = openvswitch,l2population
 
 [securitygroup]
-firewall_driver = neutron.agent.linux.iptables_firewall.OVSHybridIptablesFirewallDriver
+firewall_driver = neutron.agent.firewall.NoopFirewallDriver
```
Comment (ss7pro) [inline config/neutron/templates/ml2_conf.openvswitch.ini.j2]: We need noop here.

File: config/neutron/templates/ml2_conf.openvswitch.ini.j2 [hunk 4]
```diff
@@ -4,13 +4,17 @@ debug = true
...
 
 [agent]
 tunnel_types = vxlan
-l2_population = true
+l2_population = false
```
Comment (ss7pro) [inline config/neutron/templates/ml2_conf.openvswitch.ini.j2]: Please add a "#FIXME" comment.

File: config/nova/templates/nova.conf.j2 [hunk 5]
```diff
@@ -22,7 +23,7 @@ ec2_workers = {{ workers }}
 notification_driver = noop
 
 use_neutron = True
-security_group_api = neutron
+security_group_api = nova
```
Comment (ss7pro) [inline config/nova/templates/nova.conf.j2]: Please use neutron here.

File: config/nova/templates/nova.conf.j2 [hunk 6]
```diff
@@ -143,7 +144,7 @@ password = {{ nova_keystone_password }}
 [libvirt]
 connection_uri = "qemu+tcp://{{ get_ip_address }}/system"
 {% if ceph_enabled %}
-images_type = rbd
+images_type = qcow2
```
Comment (ss7pro) [inline config/nova/templates/nova.conf.j2]: This needs to be moved above ceph_enabled section.

File: config/glance/templates/glance-api.sh.j2 [hunk 7]
```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -e
-cp `find / -name glance-api-paste.ini` /etc/glance/
+cp `find / -not -path "/etc" -name glance-api-paste.ini` /etc/glance/
```
Comment (ss7pro) [inline config/glance/templates/glance-api.sh.j2]: add '#FIXME' comment here.

File: config/neutron/templates/ml2_conf.ini.j2 [hunk 8]
```diff
@@ -16,11 +16,13 @@ vni_ranges = 1:1000
 vxlan_group = 239.1.1.1
 
 [securitygroup]
-firewall_driver = neutron.agent.linux.iptables_firewall.OVSHybridIptablesFirewallDriver
+firewall_driver = noop
```
Comment (ss7pro) [inline config/neutron/templates/ml2_conf.ini.j2]: add '#FIXME' comment here.

File: config/neutron/templates/ml2_conf.openvswitch.ini.j2 [hunk 9]
```diff
@@ -4,13 +4,18 @@ debug = true
 mechanism_drivers = openvswitch,l2population
 
 [securitygroup]
-firewall_driver = neutron.agent.linux.iptables_firewall.OVSHybridIptablesFirewallDriver
+firewall_driver = noop
```
Comment (ss7pro) [inline config/neutron/templates/ml2_conf.openvswitch.ini.j2]: add '#FIXME' comment here.

File: config/neutron/templates/ml2_conf.openvswitch.ini.j2 [hunk 10]
```diff
@@ -4,13 +4,18 @@ debug = true
...
+# TODO(DTadrzak): FIX ME
+l2_population = false
 arp_responder = true
 
 [ovs]
 bridge_mappings = physnet1:{{ neutron_bridge_name }}
 local_ip = {{ get_ip_address }}
+
+[vxlan]
+l2_population = false
```
Comment (ss7pro) [inline config/neutron/templates/ml2_conf.openvswitch.ini.j2]: add '#FIXME' comment here.

File: config/nova/templates/nova.conf.j2 [hunk 11]
```diff
@@ -2,7 +2,8 @@
 [DEFAULT]
 debug = {{ nova_logging_debug }}
 vif_plugging_is_fatal = False
```
Comment (ss7pro) [inline config/nova/templates/nova.conf.j2]: add '#FIXME' comment here.

File: config/nova/templates/nova.conf.j2 [hunk 12]
```diff
@@ -22,7 +23,7 @@ ec2_workers = {{ workers }}
 notification_driver = noop
 
 use_neutron = True
-security_group_api = neutron
+security_group_api = nova
 network_api_class = nova.network.neutronv2.api.API
 firewall_driver = nova.virt.firewall.NoopFirewallDriver
```
Comment (ss7pro) [inline config/nova/templates/nova.conf.j2]: add '#FIXME' comment here.

File: config/neutron/templates/ml2_conf.ini.j2 [hunk 13]
```diff
@@ -16,11 +16,14 @@ vni_ranges = 1:1000
...
 [securitygroup]
-firewall_driver = neutron.agent.linux.iptables_firewall.OVSHybridIptablesFirewallDriver
+# TODO(DTadrzak): FIX ME
+firewall_driver = noop
+enable_security_group = False
 
 [agent]
 tunnel_types = vxlan
-l2_population = true
+#TODO(DTadrzak): Fix me
```
Comment (PiotrProkop) [inline config/neutron/templates/ml2_conf.ini.j2]: Please change to # TODO(DTadrzak): FIX ME

**Comment by PiotrProkop** [comment]
LGTM

**Pull request merged**